src/backbone.marionette.js

```diff
@@ -85,6 +85,9 @@ Marionette.triggerMethodMany = triggerMethodMany;
 Marionette.isEnabled = isEnabled;
 Marionette.setEnabled = setEnabled;
 
+Marionette.Behaviors = {};
```

Comment:
Yeah.. so that turned out to be more difficult than I expected with the final build.  importing Marionette works for DEV_MODE for instance because it's used inside a function, but declaring things in the global scope of the module doesn't jive well with `backbone.marionette` later.  There might be a cleaner solution, but I'm not sure atm what it is.